src/current/_includes/releases/testing-release-notice.md

```diff
@@ -1,5 +1,11 @@
-{% if include.major_version.release_date == "N/A" %}{% comment %} check if a release date has been set {% endcomment %}
+{% if include.major_version.release_type == "Testing" %}
```

Comment:
When and where would this file and content be included? I'm not seeing it pulled in via existing logic anywhere. I may be missing something.

Reply:
It's included by the version-specific release note stub pages, like https://www.cockroachlabs.com/docs/releases/v23.2. I don't need to update that file in this PR.